pkg/kubectl/bind/plugin/bind.go

```diff
@@ -279,6 +283,10 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 			}
 		})
 
+		if b.KonnectorImageOverride != "" {
+			args = append(args, "--konnector-image"+b.KonnectorImageOverride)
```

Comment:
not `"--konnector-image="+b.KonnectorImageOverride` ?

Reply:
Not actually. We are putting the flag key and value in a string slice which is handled according. So no need for '='.

Reply:
Sorry, misunderstood before. Fixed it now.